### PR TITLE
Find form in render_form tag

### PR DESCRIPTION
**Pull request by cblignaut**
@qoda: Could you please check this one ?

### CODE REVIEW

File: formfactory/templatetags/formfactory_tags.py [hunk 1]
```diff
@@ -8,5 +9,5 @@
     "formfactory/inclusion_tags/form_detail.html", takes_context=True
 )
 def render_form(context, obj):
-    context["object"] = obj
+    context["object"] = Form.objects.get_object_or_404(slug=obj)
```
Comment (qoda) [inline formfactory/templatetags/formfactory_tags.py]: You'll need to detect weather the obj passed is a slug or object and than apply the appropriate logic. Docs and tests missing.

File: formfactory/templates/formfactory/inclusion_tags/form_detail.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<form class="Form" method="POST">{% csrf_token %}
+<form class="Form" method="POST" action="{{ object.post_to|default:"" }}">{% csrf_token %}
```
Comment (hedleyroos) [inline formfactory/templates/formfactory/inclusion_tags/form_detail.html]: Not needed anymore. Merge with develop please. `action="."` is all that is required.

Comment (qoda) [inline formfactory/templates/formfactory/inclusion_tags/form_detail.html]: Shouldn't be here, `post_to` was removed. Please merge with develop.

File: formfactory/templatetags/formfactory_tags.py [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
 register = template.Library()
 
 
 @register.inclusion_tag(
     "formfactory/inclusion_tags/form_detail.html", takes_context=True
 )
-def render_form(context, obj):
-    context["object"] = obj
+def render_form(context, object_or_slug):
+    if isinstance(object_or_slug, models.Form):
```
Comment (qoda) [inline formfactory/templatetags/formfactory_tags.py]: object will be an instance of formfactory.models.Form

Comment (phalamag) [inline formfactory/templatetags/formfactory_tags.py]: Yes. That's what I am checking for.

Comment (qoda) [inline formfactory/templatetags/formfactory_tags.py]: Oh crap, sorry your right, read it wrong.

**Pull request merged**